Source/WebCore/workers/service/server/SWServer.cpp

```diff
@@ -1942,4 +1942,19 @@ void SWServer::Connection::retrieveRecordResponseBody(BackgroundFetchRecordIdent
         callback(makeUnexpected(ResourceError { errorDomainWebKitInternal, 0, { }, "No server found"_s }));
 }
 
+#if ENABLE(CONTENT_EXTENSIONS)
+void SWServer::reportNetworkUsageToAllWorkerClients(ServiceWorkerIdentifier identifier, size_t delta)
+{
+    if (RefPtr worker = workerByID(identifier)) {
+        if (RefPtr connection = contextConnectionForRegistrableDomain(worker->topRegistrableDomain())) {
+            ServiceWorkerClientQueryOptions options = { true, ServiceWorkerClientType::All };
```

Comment:
```suggestion
            ServiceWorkerClientQueryOptions options = { true, ServiceWorkerClientType::Window };
```
Since you only report for iframes.